src/unittest/test_random.cpp

```diff
@@ -91,8 +99,18 @@ void TestRandom::testPcgRandom()
 {
 	PcgRandom pr(814538, 998877);
 
-	for (u32 i = 0; i != 256; i++)
+	for (u32 i = 0; i != 128; i++)
 		UASSERTEQ(u32, pr.next(), expected_pcgrandom_results[i]);
+
+	PcgRandom pr2(814538, 998877);
```

Comment:
these values are not relevant for the test
```suggestion
	PcgRandom pr2(0, 0);
```